Age.js

```diff
@@ -0,0 +1,5 @@
+console.log("Homework-1");
+let yearOfBirth = 1990;
+let yearFuture = 2045;
+let age =  yearFuture-yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
```

Comment:
This file is a duplicate of `Age-ify.js`. You are welcome to delete one or the other.